.github/workflows/sbom.yaml

```diff
@@ -10,15 +13,23 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Download Syft
-        id: syft
-        uses: anchore/sbom-action/download-syft@v0.14.2
-
       - name: Generate SBOM from source code
-        run: "${{ steps.syft.outputs.cmd }} --config .github/edgebit/source-syft.yaml --file /tmp/sbom.syft.json ."
+        uses: eyakubovich/sbom-action@ey/add-config-input
```

Comment:
Not sure if you want to do this now but `anchore/sbom-action@v0` now has the config input so you can switch to that.
The only issue is that it uses the latest Syft which has a different config file format... at least the `catalogers:` is gone. It's been replaced by `select-catalogers`: https://github.com/anchore/syft?tab=readme-ov-file#configuration

Reply:
Yeah, I noticed that in the old PR I merged yesterday. I didn't want to deal with that change just yet.